### DISCUSSION TITLE
ver 6.0 Plan

### DISCUSSION THREAD

**Discussion by black7375** [black7375/Firefox-UI-Fix#380] [Ideas]
## Concept: Minimalism

- [x] One liner UI
- [x] Auto Hide UI
- [x] Centered UI

### Before `v6.0.0`

Add a few support for icon compatibility and miscellaneous options.

**Refactoring**
SCSS has been successfully introduced, but it is still optimized only for compatibility parts.  #234
Refactoring needs to create more concise and less duplicate code.

**Icon compatibility**
- Mac's global menu
- Linux's `widget.gtk.native-context-menus`
- Checkbox's icon #349 #375

**Miscellaneous options**
- Pinned tab's close button #378
  - `userChrome.tab.close_button_at_pinned.always`
  - `userChrome.tab.close_button_at_pinned.background`
- Dragspace at maximized https://github.com/black7375/Firefox-UI-Fix/issues/370#issuecomment-1110479254
  - `userChrome.padding.drag_space.maximized`
- Photon style's selected static separator color #141
  - `userChrome.tab.static_separator.selected_accent`

**Comment by Zabooby** [comment]
I am currently using https://github.com/crambaud/waterfall as a oneliner ui. It would be amazing if you combined that theme with the customizability of v5 like changing accent and system content colours.
